lib/modules/vm_connection.rb

```diff
@@ -16,7 +16,7 @@ class VMConnection
     ";"
 
   SYSTEM_DEPENDENCIES =
-    'export PATH="$PATH:$HOME/.rvm/bin"; source /home/miguel/.rvm/scripts/rvm'.freeze
+    'export PATH="$PATH:$HOME/.rvm/bin"; source /Users/Markus.Springer/.rvm/scripts/rvm'.freeze
```

Comment:
this looks wrong! potentially should come from secrets! and it would be nice to have the production secrets file in the encrypted in the repo.